tests/functional/spec/features/validation/sum/sum_equal_or_less_validation_against_total.spec.js

```diff
@@ -34,15 +34,15 @@ describe("Feature: Sum of grouped answers validation (equal or less than) agains
   });
 
   describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
```

Comment:
You could of just changed this to 11 and saved yourself a couple of changes😆  (definitely don't need to change now) However this has raised a bug, nothing to do with this mind. But if you get it to the end of the survey and then go back, you can change the feeding question to a value that won't validate, but because the 2nd question isn't now visited again it takes you back to summary and you can submit